spec/support/shared_examples/shared_example_for_clients.rb

```diff
@@ -91,8 +91,10 @@
             data = []
             it 'yields with a chunk, remaining length, and total length' do
               expect do
-                conn.request(method: :get, path: '/content-length/100') do |chunk, remaining_length, total_length|
-                  data = [chunk, remaining_length, total_length]
+                silence_warnings do
```

Comment:
Could we make this not warning, rather than silencing? (apologies, as I forget exactly what warning might be coming from here).

Reply:
This case is testing `when utilizing deprecated block usage` so we expect to see a warning.

Reply:
Ah, right. Thanks for the reminder.